mlrun/common/schemas/alert.py

```diff
@@ -202,6 +202,24 @@ def templates_differ(self, other):
         )
 
 
+class NotificationState(pydantic.BaseModel):
+    kind: str
+    status: Optional[str]
```

Comment:
In the requirement we approved another scheme:
```
NotificationState:
    kind: str
    err: Optional[str] #empty error means that notification was sent successfully
```

It is already used by FE to mock data and please also notice that the idea was to show the actual error, not just boolean value.